aries-site/src/data/wcag/datasearch.json

```diff
@@ -0,0 +1,512 @@
+    "label": "Target Size (Minimum)",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.1",
+    "label": "Language of Page",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.2",
+    "label": "Language of Parts",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.3",
+    "label": "Unusual Words",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.4",
+    "label": "Abbreviations",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.5",
+    "label": "Reading Level",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.1.6",
+    "label": "Pronunciation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.1",
+    "label": "On Focus",
+    "status": "conditional",
+    "notes": "when using the defaultSuggestion prop this will fail so this prop should be used with caution."
+  },
+  {
+    "rule": "3.2.2",
+    "label": "On Input",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.3",
+    "label": "Consistent Navigation",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.4",
+    "label": "Consistent Identification",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.5",
+    "label": "Change on Request",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.2.6",
+    "label": "Consistent Help",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.1",
+    "label": "Error Identification",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.2",
+    "label": "Labels or Instructions",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.3",
+    "label": "Error Suggestion",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.4",
+    "label": "Error Prevention (Legal, Financial, Data)",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.5",
+    "label": "Help",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.6",
+    "label": "Error Prevention (All)",
+    "status": "not-applicable",
+    "notes": "Dependent on FormField."
+  },
+  {
+    "rule": "3.3.7",
+    "label": "Redundant Entry",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.8",
+    "label": "Accessible Authentication (Minimum)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "3.3.9",
+    "label": "Accessible Authentication (Enhanced)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "4.1.2",
+    "label": "Name, Role, Value",
+    "status": "conditional",
+    "notes": "When not in FormField, caller must provide an aria-label or title."
```

Comment:
This component does use a FormField, however the label is only customizable at the global messages level which feels wrong

Reply:
I added a note here lmk what you think